sdk/cosmos/azure-cosmos-benchmark/src/main/java/com/azure/cosmos/benchmark/Configuration.java

```diff
@@ -61,6 +61,9 @@
     @Parameter(names = "-maxConnectionPoolSize", description = "Max Connection Pool Size")
     private Integer maxConnectionPoolSize = 1000;
 
+    @Parameter(names = "-thresholdForDiagnosticsInMs", description = "Latency threshold for printing diagnostics")
+    private Integer thresholdForDiagnosticsInMs = 60000;
```

Comment:
Could you please use Duration format instead of Milisecond here?
We will have other time configurations and some can be a few hours. Duration will be the most suitable format.

Reply:
Duration more suits in scenario like maxRunningTimeDuration, but here we have latency  threshold which will  be in seconds at max.  Integer  thresholdForDiagnosticsInMs is more intuitive here

Reply:
all the public APIs of the SDK for latency regardless of being a short duration or high duration use duration. same goes for  maxRunningTimeDuration config in the benchmark. Duration can be used for both small and long duration configuration. Please let's be consistent everywhere.

Reply:
Done, only concern was if we choose to set it under one sec then we have to pass in decimal like PT0.010S.